src/persistence/db/upgrades/dbto11.cpp

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
There's a lot of code here, but  the tests look like they ensure that the schema has upgraded successfully and the right fields are in the right spots. I'm going to consider the SQL correct based off that.